src/firebaseWrapper.js

```diff
@@ -37,9 +37,31 @@ const logout = () => {
 /**
  * Messaging Functions
  */
-// const sendMessage = (conversationId, userId, message) => {};
+const sendMessage = (user, receiver, message) => {
+  firebase
+    .database()
+    .ref(`users/${user}/friends/${receiver}/`)
+    .on("value", snapshot => {
+      const conversationId = snapshot.val();
+      firebase
+        .database()
+        .ref(`/conversations/${conversationId}/messages/`)
+        .push(message);
+    });
+};
 // const getAllConversations = userId => {};
-// const listenForMessages = conversationId => {};
+const listenForMessages = (updateMessages, user, receiver) => {
```

Comment:
could you actually make the interface for this `(conversationId, done) => {}`? In later stages, we'll have access to the conversationIds more easily than the user and receiver Ids i think. You can hard code the conversation Id to whatever the conversation Id is between those two users in the firebase.